src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderProgramProfile.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Data
+@EqualsAndHashCode(callSuper=false)
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "OFFENDER_PROGRAM_PROFILES")
+@ToString(of = {"offenderProgramReferenceId"})
+public class OffenderProgramProfile extends ExtendedAuditableEntity {
+
+    @Id
+    @Column(name = "OFF_PRGREF_ID")
+    private Long offenderProgramReferenceId;
+
+    @ManyToOne(optional = false, fetch = FetchType.LAZY)
+    @JoinColumn(name = "OFFENDER_BOOK_ID", nullable = false)
+    private OffenderBooking offenderBooking;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    // TODO - @NotFound(action = IGNORE)
+    @JoinColumn(name = "CRS_ACTY_ID")
```

Comment:
maybe check if there is a foreign key in NOMIS and assuming not is there any bad data that would cause this to fail?

Reply:
Ah yes. I am fixing a naming issue, so i'll sort this out